.circleci/config.yml

```diff
@@ -10,6 +10,8 @@ workflows:
         - test-2.070-prod
         - test-2.071-devel
         - test-2.071-prod
+        - test-2.077-devel
```

Comment:
I'd name it `test-2.077-upstream-{prod|devel}`, just to make it more obvious in reported github status